.gitignore

```diff
@@ -22,4 +22,5 @@ ligradle/
 .DS_Store
 _site/
 .sass-cache/
+.jekyll-cache/
```

Comment:
what's the purpose to add this one?

Reply:
I'm assuming this shouldn't be included in the source code. It only popped up now because it's new Jekyll functionality, I guess.